src/frontend/src/views/AgreeTerms/index.tsx

```diff
@@ -68,7 +68,7 @@ export default function AgreeTerms(): JSX.Element {
                 key things to know:
               </ListSubheader>
               <ListItem fontSize="s">
-                You {" "}<B> always </B>{" "} own and control the data you
+                You &nbsp;<B>always</B>&nbsp; own and control the data you
```

Comment:
I'm wary of using non-breaking spaces for anything other than exactly that, and even then I'm usually worried they'll bite me. If the primary reason is to increase padding on bold font, then I probably made the wrong suggestion with switching over to the general `B` component instead of a custom BoldText. You could toss an additional style on the `B` in this file:
```
import styled from "@emotion/styled";

...

const spacedBold = styled(B)`
  ${(props) => {
    const spacings = getSpacings(props);
    return `
      padding: 0 ${spacings?.s}px;
    `;
  }}
`;
```

Would probably be the most matched to how things are done elsewhere. Although I think it would also be pretty reasonable to just have
```
styled(B)`
  padding: 0 0.5em; // adjust 0.5em size to taste, but because em, will handle later font size changes well
`;
```